### PR TITLE
docs: Update why-uno-platform.md

### PR DESCRIPTION
**Pull request by sasakrsmanovic**
Brushed up two answers

GitHub Issue (If applicable): closes #

## PR Type
- Documentation content changes

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: doc/articles/concepts/overview/why-uno-platform.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ The Windows Community Toolkit (WCT) is a collection of helper functions, control
 
 ## More UI controls with MAUI Embedding
 
-Uno Platform allows for embedding .NET MAUI-specific controls from all leading 3rd party vendors like Syncfusion, Grial Kit, Telerik, DevExpress, Esri, Grape City, and the .NET MAUI Community Toolkit. Keep in mind that this cross-platform approach works only for target platforms .NET MAUI reaches – iOS, Android, macOS, and Windows. [We have 6 sample apps](xref:Uno.Extensions.Maui.Overview) showing how to work with MAUI embedding by all leading 3rd party vendors.
+Uno Platform allows for embedding .NET MAUI-specific controls from all leading 3rd party vendors like Syncfusion, Grial Kit, Telerik, DevExpress, Esri, Grape City, and the .NET MAUI Community Toolkit. Keep in mind that this cross-platform approach works only for target platforms .NET MAUI reaches – iOS, Android, macOS, and Windows. [We have 6 sample apps](xref:Uno.Extensions.Maui.Overview) showing how to work with MAUI embedding by all leading 3rd party vendors. Note: Uno Platform is not built on top of on .NET MAUI.
```
Comment (Youssef1313) [inline doc/articles/concepts/overview/why-uno-platform.md:106]:
```suggestion
Uno Platform allows for embedding .NET MAUI-specific controls from all leading 3rd party vendors like Syncfusion, Grial Kit, Telerik, DevExpress, Esri, Grape City, and the .NET MAUI Community Toolkit. Keep in mind that this cross-platform approach works only for target platforms .NET MAUI reaches – iOS, Android, macOS, and Windows. [We have 6 sample apps](xref:Uno.Extensions.Maui.Overview) showing how to work with MAUI embedding by all leading 3rd party vendors.

> [!NOTE]
> Uno Platform is not built on top of on .NET MAUI.
```

File: doc/articles/concepts/overview/why-uno-platform.md [hunk 2]
```diff
@@ -222,7 +222,7 @@ Uno Platform supports SQLite for all platforms, including WebAssembly, to allow
 
 ## Proven migration path for Xamarin.Forms
 
-In addition to providing 3rd party control MAUI Embedding, Uno Platform is a proven modernization path for Xamarin.Forms apps. We've collated numerous [docs](https://platform.uno/xamarin.forms) to aid you in the migration. Big brands such as **TOYOTA** have already migrated some of their apps to Uno Platform.
+In addition to providing 3rd party control MAUI Embedding, Uno Platform is a proven modernization path for Xamarin.Forms apps. We've collated numerous [docs](https://platform.uno/xamarin.forms) to aid you in the migration. Big brands such as **TOYOTA** have already migrated some of their apps to Uno Platform. Note: Uno Platform is not built on top of on .NET MAUI.
```
Comment (Youssef1313) [inline doc/articles/concepts/overview/why-uno-platform.md:225]:
```suggestion
In addition to providing 3rd party control MAUI Embedding, Uno Platform is a proven modernization path for Xamarin.Forms apps. We've collated numerous [docs](https://platform.uno/xamarin.forms) to aid you in the migration. Big brands such as **TOYOTA** have already migrated some of their apps to Uno Platform.

> [!NOTE]
> Uno Platform is not built on top of .NET MAUI.
```

**Review by Youssef1313 — APPROVED** [review]

**Review by jeromelaban — APPROVED** [review]

**Comment by jeromelaban** [comment]
/

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.1

**Pull request merged**